bc_obps/common_utils/email/email_service.py

```diff
@@ -0,0 +1,88 @@
+import json
+from datetime import datetime, timedelta
+from typing import List
+from uuid import UUID
+
+import requests
+from django.conf import settings
+
+
+class EmailService:
+    def __init__(self):
+        self.api_url = settings.CHES_API_URL
+        self.client_id = settings.CHES_CLIENT_ID
+        self.client_secret = settings.CHES_CLIENT_SECRET
+        self.token_endpoint = settings.CHES_TOKEN_ENDPOINT
+        self.token = None
```

Comment:
nit: We could use some type hints here? for instance: `self.token: Optional[str] = None`

Reply:
done :)